apps/web/app/(auth)/auth/login/components/SigninForm.tsx

```
@@ -95,6 +95,7 @@ export const SigninForm = ({
     }
   };
 
+  const [lastLogin, setLastLogin] = useState<String | null>("");
```

Biome

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)